user/views.py

```diff
@@ -89,3 +89,17 @@ def post(self, request):
         except KeyError:
             return JsonResponse({"message":"INVALID_KEYS"}, status = 400)
 
+class CheckSignInIdView(View): 
+        def post(self, request):
+            data = json.loads(request.body)
+            try:
+                login_id = data.get('login_id', None)
+                email    = data.get('email', None)
+
+                if User.objects.filter(email = email).exists() or User.objects.filter(login_id = login_id).exists():
+                    return HttpResponse(status = 200)
+
+                return JsonResponse({"message": "not existing account"}, status = 400)
```

Comment:
계정정보가 없다고 체크된 정상적인경우라서 200으로 수정부탁드립니다.
```suggestion
                return JsonResponse({"message": "not existing account"}, status = 200)
```